federation/cluster/common.sh

```diff
@@ -87,7 +87,18 @@ function create-federation-api-objects {
       fi
     fi
 
-    export FEDERATION_SERVICE_CIDR=${FEDERATION_SERVICE_CIDR:-"10.10.0.0/24"}
+    if [[ "${KUBERNETES_PROVIDER:-}" == "gce" || "${KUBERNETES_PROVIDER:-}" == "gke" ]]; then
+      PD_PLUGIN_NAME='kubernetes.io/gce-pid'
```

Comment:
@quinton-hoole In the alpha version of dynamic provisioning, provisioners are hard coded based on cloud provider (see [this](https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-controller-manager/app/plugins.go#L110)). Moving forward, however, there will not necessarily be a 1:1 coupling between cloud provider and provisioner, see https://github.com/kubernetes/kubernetes/pull/26908

CC @pmorie